code/modules/paperwork/paper.dm

```diff
@@ -372,7 +372,7 @@
tc. Essentially, they are the maintenance & expansion crew.<BR>1. Maintain the Station<BR>2. Expand the Station<BR>3. Keep Power Running<BR><BR><B>Atmospheric Technician</B><BR>Atmospheric Technician's are responsible for the atmospherics. They keep the station hot, cold, and breathable.<BR>1. Keep the air inside the Station.<BR>2. Keep the Station warm.<BR>3. Extinguish fires.<BR><BR><B>Scientist</B><BR>Scientists research advancements in technology. They keep us up-to-date in modern times.<BR>1. Research Technology.<BR>2. Make Bombs.<BR>3. Breed Slimes.<BR><BR><B>Roboticist</B><BR>Roboticists are scientists specializing in cyborgs, mechs, and drones. They maintain cyborgs and build mechs.<BR>1. Maintain & Create Cyborgs<BR>2. Build & Maintain Mechs<BR>3. Mass-Produce Medibots<BR><BR><B>Medical Doctor</B><BR>Medical Doctors are responsible for keeping the crew happy, healthy, and alive. They answer to the Chief Medical Officer.<BR>1. Perform Surgeries<BR>2. Heal Injuries<BR>3. Drug the Unhealthy.<BR><BR><B>Chemist</B><BR>Chemists are responsible for creating drugs for use by the Station.<BR>1. Create Drugs<BR>2. Create Cleaning Fluid<BR><BR><B>Geneticist</B><BR>Geneticists are responsible for manipulating the human/monkey/etc. Genome and cloning the deceased. They are playing God.<BR>1. Research Genetics<BR>2. Clone the Deceased<BR><BR><B>Virologist</B><BR>The Virologist is responsible for creating beneficial viruses and curing harmful ones. They are the pathologists of our crew.<BR>1. Create Beneficial Viruses<BR>2. Cure Harmful Diseases<BR><BR><B>Quartermaster</B><BR>The Quartermaster manages Supply, directly under the Head of Personnel's supervision. They keep the station supplied and manage distribution of materials and minerals.<BR>1. Manage Cargo<BR>2. Direct Miners<BR>3. Distribute Supplies<BR><BR><B>Cargo Technician</B><BR>Cargo Technicians operate Cargo's supply requests, move objects on and off the shuttle, and distribute goods personally.<BR>1. Approve & Deny Supply Requests<BR>2. Load & Unload the Cargo Shuttle<BR>3. Distribute Supplies<BR><BR><B>Shaft Miner</B><BR>Shaft Miners mine and refine resources such as plasma, diamonds, and iron for the station's use in robotics, chemistry, and research.<BR>1. Mine Resources<BR>2. Refine Resources<BR><BR><B>Janitor</B><BR>Janitors are in charge of keeping the station clean and appearing well maintained.<BR>1. Clean the Station<BR>2. Dispose of garbage<BR><BR><B>Bartender</B><BR>The Bartender is in charge of providing the crew with alcoholic & non-alcoholic beverages. When people need to drown their sorrows, they turn to him.<BR>1. Give Drink to the Thirsty<BR>2. Keep Peace in the Bar<BR><BR><B>Cook</B><BR>The Cook is in charge of preparing fine meals to feed the Station's crew.<BR>1. Give Food to the Hungry<BR><BR><B>Botanist</B><BR>The Botanist is in responsible for the growth of plants for the Cook to use in meal preparation and for raw consumption by the populous.<BR>1. Grow Food<BR><BR>"
```

Comment:
- You missed adding a `<br>` right after `<b>Security Officer</b>`.
- Punctuation is inconsistent in the numbered lists. Some end with a period, but most don't. Make it all or nothing.
- Atmos techs have "They keep the station hot, cold, and breathable." which sounds weird and contradicts itself.
- The geneticist one has a rogue period after the /etc in the description.
- The Botanist one has a misspelling: `populace` not `populous`.